custom/enikshay/tests/test_case_utils.py

```diff
@@ -4,12 +4,15 @@
 from django.test import TestCase
 from custom.enikshay.tests.utils import ENikshayCaseStructureMixin
 from corehq.form_processor.tests.utils import run_with_all_backends, FormProcessorTestUtils
+from casexml.apps.case.models import CommCareCase
 
 from custom.enikshay.case_utils import (
     get_open_episode_case_from_person,
     get_adherence_cases_between_dates,
     get_occurrence_case_from_episode,
     get_person_case_from_occurrence,
+    get_person_case_from_episode,
+    _update_case
```

Comment:
since we are accessing this from other modules, I'd rename it to be non-private `update_case`